test/packs/fake_v2.ts

```diff
@@ -121,20 +121,24 @@ pack.addSyncTable({
 pack.addDynamicSyncTable({
   name: 'FooDyns',
   description: 'FooDynDesc',
-  getName: coda.makeMetadataFormula(async () => 'name'),
-  getDisplayUrl: coda.makeMetadataFormula(async () => 'display url'),
-  getSchema: coda.makeMetadataFormula(async () => {
-    return coda.makeSchema({
-      type: coda.ValueType.Array,
-      items: {type: coda.ValueType.Object, properties: {}},
-    });
-  }),
+  getName: async () => 'name',
+  getDisplayUrl: async () => 'display url',
+  getSchema: async () => {
+    return {
+      type: coda.ValueType.Object,
```

Comment:
Is there also a test case that returns the array version?  Want to make sure we're exercising both.

Reply:
let me see if I can add a unittest instead.